envs/monkey_zoo/blackbox/utils/gcp_machine_handlers.py

```diff
@@ -68,19 +68,18 @@ def get_set_project_command(project):
     return SET_PROPERTY_PROJECT % project
 
 
-def run_gcp_command(arglist):
-    gcp_cmd, machine_list, zone = arglist
+def _run_gcp_command(gcp_command: str, machine_list: Collection[str], zone: str):
+    """Runs the command in the given zone"""
     ret = subprocess.run(  # noqa DUO116
-        (gcp_cmd % (" ".join(machine_list), zone)),
+        (gcp_command % (" ".join(machine_list), zone)),
         stderr=subprocess.PIPE,
         stdout=subprocess.PIPE,
         shell=True,
     )
     if ret.returncode != 0:
-        raise Exception(f"Failed starting GCP machines: {ret.stderr}")
+        raise Exception(f"Failed starting GCP machines: {ret.stderr.decode()}")
 
 
-def run_gcp_pool(gcp_command, machine_list):
-    arglist = [(gcp_command, machine_list[zone], zone) for zone in machine_list]
-    with Pool(2) as pool:
-        pool.map(run_gcp_command, arglist)
```

Comment:
This also worked in the past?

Reply:
I updated the version of gcloud CLI on the island machines, so that may have contributed to this change. It didn't manifest as a problem on Linux though. Here's [one of the runs](https://jenkins.guardi/view/Monkey/job/run_msi_ete/1149/console) where the problem occurred

Reply:
Using the release of gcloud CLI with python bundled resolves the problem with parallel invocations. Interestingly, it's not any faster than running serially, at least on Windows.

I also updated this section of code to use a pool size that matches the number of commands that are run